.buildkite/pipelines/esql_grammar_sync.yml

```diff
@@ -17,3 +17,12 @@ steps:
       provider: gcp
       machineType: n2-standard-2
       preemptible: true
+  - command: .buildkite/scripts/steps/esql_generate_function_docs.sh
+    label: Generate Function Definitions
```

Comment:
Can we change the label to indicate that this is for docs and not for functions?